manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -15,7 +15,14 @@ metadata:
 webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
-      url: https://{{ .Values.sidecarInjectorWebhook.remoteWebhookHost }}:{{ .Values.sidecarInjectorWebhook.remoteWebhookPort }}/inject
+      {{- if and (.Values.global.remotePilotAddress) (.Values.global.centralIstiod) }}
```

Comment:
should this file be moved to `manifests/charts/base/templates`, so that it's next to the `validatingwebhookconfiguration.yaml`?

Reply:
@costinm @howardjohn didn't want to move it - we had a very length discussion in one of the env WG mtgs, the net is because we handle different revision for mutatingwebhook configs.

Reply:
Yes, short term it has to stay here.